### PR TITLE
add orders for admin

### PR DESCRIPTION
**Pull request by kokosko**

### CODE REVIEW

File: app/assets/javascripts/orders.coffee [hunk 1]
```diff
@@ -23,6 +23,14 @@ sum = ->
       else
         $("#discount").hide()
   $("#order-sum").html(val.toFixed(2) + " руб.")
+  $("#order-sum-form").val(val.toFixed(2))
+
+sum2 = ->
```
Comment (galulex) [inline app/assets/javascripts/orders.coffee]: @kokosko ?

File: app/controllers/admin/orders_controller.rb [hunk 2]
```diff
@@ -1,3 +1,32 @@
 class Admin::OrdersController < ApplicationController
+  before_action :set_order, only: [:edit, :update, :destroy]
   layout 'admin'
+
+  helper_method :for_oredr
```
Comment (galulex) [inline app/controllers/admin/orders_controller.rb]: Misspelling + unexciting method

File: app/controllers/admin/orders_controller.rb [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+
+  def index
+    @orders = Order.order(status: :desc, created_at: :desc).page(params[:page]).per(5)
```
Comment (galulex) [inline app/controllers/admin/orders_controller.rb]: per(20)

File: app/controllers/orders_controller.rb [hunk 4]
```diff
@@ -19,11 +18,6 @@ def create
...
   end
 
-  def destroy
-    @order.destroy
-    redirect_to orders_url
-  end
-
   private
 
   def set_order
```
Comment (galulex) [inline app/controllers/orders_controller.rb]: Should be removed as well

File: app/views/admin/orders/index.html.slim [hunk 5]
```diff
@@ -1,18 +1,36 @@
...
+            p = order.name
+            p = order.phone
+            p = order.adress
+            p = order.email
+          td
+            p #{ t('order.sum') }: #{number_to_currency(order.sum, locale: I18n.locale) }
+            p #{ t('discount.title') }: #{ number_to_currency(order.discount, locale: I18n.locale) }
+            p #{ t('order_sum') }: #{ number_to_currency(order.total, locale: I18n.locale) }
+          td
+            p = order.closed? ? t('order.closed') : t('order.active')
```
Comment (galulex) [inline app/views/admin/orders/index.html.slim]: you can string interpolation here. But the best case is to use `enum` localization technique

File: app/assets/javascripts/orders.coffee (hunk 1 above)
Comment (kokosko) [inline app/assets/javascripts/orders.coffee]: #order-sum shows for user
#order-sum-form - hidden field in form

File: app/controllers/admin/orders_controller.rb (hunk 2 above)
Comment (kokosko) [inline app/controllers/admin/orders_controller.rb]: fixed

File: app/controllers/admin/orders_controller.rb (hunk 3 above)
Comment (kokosko) [inline app/controllers/admin/orders_controller.rb]: fixed

File: app/assets/javascripts/orders.coffee (hunk 1 above)
Comment (kokosko) [inline app/assets/javascripts/orders.coffee]: delete sum2

File: app/controllers/orders_controller.rb (hunk 4 above)
Comment (kokosko) [inline app/controllers/orders_controller.rb]: fixed

**Pull request merged**